src/components/UI/AcceptBar.jsx

```diff
@@ -0,0 +1,64 @@
+import * as React from 'react'
+import Switch from '@mui/material/Switch'
+import { FormControlLabel, Grid } from '@mui/material'
+import styled from '@emotion/styled'
+
```

Comment:
<img width="451" alt="Screenshot 2023-04-06 at 13 29 55" src="https://user-images.githubusercontent.com/44497623/231983760-ecff6b3f-9d63-4b65-a0d1-c4b4f8c27aff.png">
why file and component name are different ?